conformance_pack/rds.sp

```diff
@@ -232,6 +252,16 @@ control "rds_db_instance_automatic_minor_version_upgrade_enabled" {
   })
 }
 
+control "rds_db_cluster_deletion_protection_enabled" {
+  title       = "RDS clusters should have deletion protection enabled"
+  description = "This control checks whether RDS clusters have deletion protection enabled. This control is intended for RDS DB instances. However, it can also generate findings for Aurora DB instances, Neptune DB instances, and Amazon DocumentDB clusters. If these findings are not useful,then you can suppress them."
```

Comment:
As per the description, does the AWS config evaluate RDS DB instances as well? Does it also evaluate Neptune DB instances, and Amazon DocumentDB clusters?